project/settings/middlewares.py

```diff
@@ -1,12 +1,12 @@
 # -*- coding: utf-8 -*-
 
 
-MIDDLEWARE_CLASSES = (
+MIDDLEWARE = (
+    'corsheaders.middleware.CorsMiddleware',
     'django.contrib.sessions.middleware.SessionMiddleware',
     'django.middleware.common.CommonMiddleware',
     'django.middleware.csrf.CsrfViewMiddleware',
     'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.auth.middleware.SessionAuthenticationMiddleware',
```

Comment:
А почему эту убираем?

Reply:
The SessionAuthenticationMiddleware class is removed. It provided no functionality since session authentication is unconditionally enabled in Django 1.10